test/pkg/environment/environment.go

```diff
@@ -27,13 +25,15 @@ import (
 )
 
 var ClusterName = flag.String("cluster-name", env.WithDefaultString("CLUSTER_NAME", ""), "Cluster name enables discovery of the testing environment")
-var Region = flag.String("region", env.WithDefaultString("AWS_REGION", ""), "Region that your test cluster lives in.")
+
+// Region uses AWS_DEFAULT_REGION which is preferred over AWS_REGION https://github.com/aws/aws-sdk-go/issues/2103
```

Comment:
were you seeing issues with this? 

https://github.com/aws/aws-sdk-go/issues/2103#issuecomment-413652038

In general, I think using `AWS_REGION` is generally better since it's been around longer and in the CLI overrides `AWS_DEFAULT_REGION`.  The aws-go-sdk also notes to use `AWS_REGION` https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#:~:text=Set%20the%20AWS_REGION%20environment%20variable%20to%20the%20default%20Region

Reply:
Oh I guess your code is actually correct, the comment just doesn't make sense?

Reply:
I ripped all of this out :P. Turns out the SDK *just works* if you have AWS_DEFAULT_REGION set.